packages/framework/src/Component/UploadedFile/UploadedFileLocator.php

```diff
@@ -13,25 +14,28 @@ class UploadedFileLocator
     protected $uploadedFileDir;
 
     /**
-     * @var string
+     * @var \League\Flysystem\FilesystemInterface
      */
-    protected $uploadedFileUrlPrefix;
+    protected $filesystem;
 
     /**
-     * @var \League\Flysystem\FilesystemInterface
+     * @var \Shopsys\FrameworkBundle\Component\Router\DomainRouterFactory
      */
-    protected $filesystem;
+    protected $domainRouterFactory;
 
     /**
      * @param string $uploadedFileDir
-     * @param string $uploadedFileUrlPrefix
      * @param \League\Flysystem\FilesystemInterface $filesystem
+     * @param \Shopsys\FrameworkBundle\Component\Router\DomainRouterFactory $domainRouterFactory
      */
-    public function __construct($uploadedFileDir, $uploadedFileUrlPrefix, FilesystemInterface $filesystem)
-    {
+    public function __construct(
+        $uploadedFileDir,
```

Comment:
Is here any reason for missing typ hint?
```suggestion
        string $uploadedFileDir,
```

Reply:
class is not using strict_types and I was not updating that part of code :)

Reply:
Could not it be fixed?

Reply:
Yes my lord